Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: all rockspec build clean
 
-VERSION=1.0.0
+VERSION=1.1.2
```

Comment:
Can do, I had to make a release for myself earlier, hence the major and patch. Will change.